src/bilder/images/dagster/templates/docker-compose.yaml.j2

```diff
@@ -72,3 +73,23 @@ services:
       - "traefik.http.middlewares.traefik-forward-auth.forwardauth.address=http://traefik-forward-auth:4181"
       - "traefik.http.middlewares.traefik-forward-auth.forwardauth.authResponseHeaders=X-Forwarded-User"
       - "traefik.http.services.traefik-forward-auth.loadbalancer.server.port=4181"
+  # per node healthcheck container
+  health-check:
+    image: 610119931565.dkr.ecr.us-east-1.amazonaws.com/dockerhub/mitodl/ol-infra-health-checks:latest
+    container_name: health-check
+    ports:
+    - "8097:8907"
+    # volumes:
+    # do I need this? Didn't need to specify when testing locally with ol-data-platform
```

Comment:
No you probably don't need these from looking at https://github.com/mitodl/ol-infra-health-checks